components/badge.css

```diff
@@ -0,0 +1,51 @@
+/* resets browser defaults */
+@import "./reset.css";
+
+/* adds basic styling */
+@import "./style-guide.css";
+
+/* BADGE */
+
+/* common style */
+.badge {
+    background: var(--primary);
+    color: var(--background-grey);
+    display: inline-block;
+    padding: 0.25em 0.4em;
+    font-size: 75%;
+    font-weight: var(--bold);
+    border-radius: 0.25rem;
+    line-height: 1;
+    vertical-align: baseline;
+    text-align: center;
+}
+
+  /* --primary: #6002EE;
```

Comment:
remove commented code

Reply:
fixed in [0b82f92](https://github.com/DivyanshSareen/PeaceUI/pull/1/commits/0b82f92e9b4eb2f8b6f9685c1536d16b714c8fd7)